app/questionnaire/placeholder_renderer.py

```diff
@@ -38,8 +38,9 @@ class PlaceholderRenderer:
     strings
     """
 
-    def __init__(self, language, answer_store=None, metadata=None):
+    def __init__(self, language, schema, answer_store=None, metadata=None):
```

Comment:
Given placeholder parser doesn't require schema, it shouldn't be required here should it?